bundles/SyliusReviewBundle/index.rst

```diff
@@ -0,0 +1,19 @@
+SyliusReviewBundle
+==================
+
+*SyliusReviewBundle* is a flexible and easy-to-manager reviews system for Symfony2 applications.
+
+With this bundle you can configure what and how can be reviewed in your application. Each review consist of rating, title and comment, which allows to express opinion fully. Reviews can be confirmed or rejected
```

Comment:
@lchrusciel sadly, I think "rating" is better word:
- **rate** - a measure, quantity, or frequency, typically one measured against some other quantity or measure
- **rating** - a classification or ranking of someone or something based on a comparative assessment of their quality, standard, or performance